### ISSUE TITLE
Check animations, some likely need prefers-reduced-motion media queries

### ISSUE THREAD

**Issue by Stephen-ONeil** [TBS-EACPD/infobase#1412]
I _think_ our JS library based animations (`react-transition-group`, `d3`) respect prefers reduced motion settings. Double check that.

Otherwise, I think our loading spinner and possible some other small animations and transitions might be missing support.
